cpp/include/ucxx/worker.h

```diff
@@ -245,6 +245,23 @@ class Worker : public Component {
    */
   void initBlockingProgressMode();
 
+  /**
+   * @brief Get the epoll file descriptor associated with the worker.
+   *
+   * Get the epoll file descriptor associated with the worker when running in blocking mode.
+   * The worker only has an associated epoll file descriptor after
+   * `initBlockingProgressMode()` is executed.
+   *
+   * The file descriptor is destroyed as part of the `ucxx::Worker` destructor, thus any
+   * reference to it shall not be used after that.
```

Comment:
**polish/suggestion:** How about we reset the `_epollFileDescriptor` slot to `0` in `~Worker` after calling `close` on it. That way it is obvious if someone attempts to use a bad object.

Reply:
I think that's OK. In Python we always go through the property which needs a live worker to access things. I agree that the user should be on the hook for managing the lifetime of arbitrary untyped pointer-like handles and making sure invariants are satisfied, so nothing to do here.